data/cloudinary

```diff
@@ -0,0 +1,2 @@
+cloudinary.com
+cloudinary.net
```

Comment:
This root domain itself could not be resolved, can you provide any sub-domain in-used of `cloudinary.net` for verifying?

Reply:
cldmo.mo.cloudinary.net

See also: https://who.is/whois/cloudinary.net